### ISSUE TITLE
Person biography not working

### ISSUE THREAD

**Issue by elenado01** [cinemagoer/cinemagoer#462]
#### Issue description

*I'm trying to obtain the biography of a person, but all of them raise an error. It is strange, since i ran the same code a few months ago and the biographies from the actors were available. Does anyone know if it is a general issue with the package or is it just me?*

#### Version of Cinemagoer, Python and OS

**NOTICE: please always try the latest version from the repository before submitting a bug.**

- **Python:** `python3 -V` or, if you are using Python 2, `python -V`
- **Cinemagoer:** `python3 -c 'import imdb ; print(imdb.VERSION)'` or, if you are using Python 2, `python -c 'import imdb ; print(imdb.VERSION)'`
- **OS:** `python -c 'import platform ; print(platform.uname())'`

#### Steps to reproduce the issue

*if possible, provide a minimal code to reproduce the problem*

```python
#!/usr/bin/env python
from imdb import Cinemagoer
# create an instance of the Cinemagoer class
ia = Cinemagoer()

person = ia.get_person('0000401')
person['biography']
```

#### What's the expected result?

- before the infoset2keys showed this: 
{'main': ['headshot', 'filmography', 'imdbID', 'name'],
 'biography': ['headshot',
  'nick names',
  'birth name',
  'height',
  'mini biography',
  'trade mark',
  'trivia',
  'quotes',
  'salary history',
  'birth date',
  'birth notes']}

#### What's the actual result?

- {'main': ['headshot', 'filmography', 'imdbID', 'name'], 'biography': []}

#### Additional details

-

**Comment by DirkWuerdemann** [comment]
Hi all, I have the same Issue. When I read the the actress nm0726257 Katja Riemann I got in the last Version (2022.12.04) a Key "mini biography" and also values for "trivia". This is not available any more in version 2023.05.01. Same for "birth date", "birth name" and "birth notes"

**Comment by alberanid** [comment]
Yep, I can confirm that they changed again and again and again the format of the web pages.

I'm in a hurry, so I've just committed a quick fix for just for 'mini biography'.

Please upgrade with `pip install git+https://github.com/cinemagoer/cinemagoer` (be sure that imdb.VERSION is '2023.07.01')

Sooner or later I'll try to fix all the other information.

**Comment by txtsd** [comment]
I took a look at the code in `personParser.py` but the XPaths and related piculet-specific code looks non-trivial to rectify for a person reading it for the first time.

Birth and Death dates, specifically, are not stored in time elements anymore, but are two `li>a`s.

**Comment by alberanid** [comment]
This issue is no longer relevant due to issue #557: ss of June 2026, Cinemagoer no longer analyzes web pages and can only rely on non-commercial datasets.
